packages/odoo/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@openfn/language-odoo",
+  "version": "3.0.1",
+  "description": "The standard odoo for building OpenFn adaptors",
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
+      "require": "./dist/index.cjs"
+    },
+    "./package.json": "./package.json"
+  },
+  "scripts": {
+    "build": "pnpm clean && build-adaptor odoo",
+    "test": "mocha --experimental-specifier-resolution=node --no-warnings",
+    "test:watch": "mocha -w --experimental-specifier-resolution=node --no-warnings",
+    "clean": "rimraf dist types docs",
+    "pack": "pnpm pack --pack-destination ../../dist",
+    "lint": "eslint src"
+  },
+  "author": "Open Function Group",
+  "license": "LGPLv3",
+  "files": [
+    "dist/",
+    "types/",
+    "ast.json",
+    "configuration-schema.json"
+  ],
+  "dependencies": {
+    "@openfn/language-common": "workspace:*",
+    "odoo-await": "^3.4.1"
+  },
+  "devDependencies": {
+    "@openfn/buildtools": "workspace:^1.0.2",
```

Comment:
I think these dev dependencies can be updated. I don't think we need buildtools, simple-ast. Maybe check against intuit's dependencies?